<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDisplayDlg</class>
 <widget class="QDialog" name="CDisplayDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>298</width>
    <height>172</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Display Setup</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>130</y>
     <width>171</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QComboBox" name="fftSizecomboBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>FFT Size</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="fftAvespinBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>FFT Averaging</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="ClickResolutionspinBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>10</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> Hz</string>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>100000</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>100</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Mouse Click Resolution</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSpinBox" name="MaxDisplayRatespinBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> Updates/Sec</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>50</number>
   </property>
   <property name="singleStep">
    <number>1</number>
   </property>
   <property name="value">
    <number>20</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>80</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Max Display Rate</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxTestBench">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>121</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>UseTestBench</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="Screen2DSizespinBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> %</string>
   </property>
   <property name="minimum">
    <number>10</number>
   </property>
   <property name="maximum">
    <number>90</number>
   </property>
   <property name="singleStep">
    <number>10</number>
   </property>
   <property name="value">
    <number>50</number>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>121</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>2D Screen Size %</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CDisplayDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CDisplayDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>ActivateTestBench()</slot>
 </slots>
</ui>
